djangocms_unitegallery/cms_plugins.py

```diff
@@ -18,9 +18,15 @@ class GalleryPlugin(CMSPluginBase):
     model = Gallery
     name = _('Gallery')
     module = _('Unite gallery')
-    render_template = 'djangocms_unitegallery/gallery.html'
+#    render_template = self.get_template()
     inlines = [PhotoInline, ]
 
+    def get_render_template(self, context, instance, placeholder):
+        default = 'djangocms_unitegallery/gallery.html'
+        if CONFIG['USE_EASYTHUMBNAILS']:
```

Comment:
Instead of a setting variable, what about trying to detect the thumbnail library by importing it in a try/except ImportError ? eg:
```
try:
    import easy_thumbnails
    return 'djangocms_unitegallery/easythumb-gallery.html'
except ImportError:
    return 'djangocms_unitegallery/gallery.html'
```